packages/components/src/DateTimePickers/pickers/TimePicker/TimePicker.component.js

```diff
@@ -1,85 +1,85 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import IncrementableScrollList from '../IncrementableScrollList';
-import twoDigits from '../../shared/utils/format/twoDigits';
+import classNames from 'classnames';
+import uuid from 'uuid';
+import DebounceInput from 'react-debounce-input';
 
-// All times in this component represents a number of minutes since the beginning of the day
+import theme from './TimePicker.scss';
 
-const maxTime = 24 * 60;
+const HOURS = 'HOURS';
+const MINUTES = 'MINUTES';
 
-function getInitialTime(selectedTime) {
-	if (selectedTime === undefined) {
-		const now = new Date();
-		return now.getHours() * 60 + now.getMinutes();
+function strToNumber(value) {
+	if (value) {
+		const num = Number(value);
```

Comment:
please make the cast in the inputdatetimepicker as this removes the 0 when typing 12:02.